gcloud/storage/blob.py

```diff
@@ -149,10 +149,10 @@ def public_url(self):
             bucket_name=self.bucket.name,
             quoted_name=quote(self.name, safe=''))
 
-    def generate_signed_url(self, expiration, method='GET',
-                            client=None, credentials=None,
-                            response_type=None, response_disposition=None,
-                            generation=None):
+    def generate_signed_url(self, expiration, method='GET', client=None,
+                            content_type="application/octet-stream",
```

Comment:
Is there a reason to use this as the default? The method it calls `generate_signed_url` has `None` as the default and ISTM that it would be a suitable default here.